components/raftstore/src/store/local_metrics.rs

```diff
@@ -385,7 +385,6 @@ pub struct RaftMetrics {
     pub message_dropped: RaftMessageDropMetrics,
     pub propose: RaftProposeMetrics,
     pub process_ready: LocalHistogram,
-    pub append_log: LocalHistogram,
```

Comment:
How about reusing the metrics for writing raft logs in async write?

Reply:
Done.

Reply:
I think it may introduce some confusion.🤔